### PR TITLE
Build Census Household Accommodation

### PR DESCRIPTION
**Pull request by bitdivision**
### What is the context of this PR?
[trello](https://trello.com/c/5V7lUw1C/3184-introduce-hh-schema-build-accommodation-eng-wls-m)

This PR adds a household schema and adjusts the build process for it.

Currently the build_schema script will error because the NI household schema does not exist. I'm not sure what the best solution is here. I didn't want to copy all of the blocks / schema over.

I have included a built version in `data/en` this will be deleted before merging, it is only there to help with quick-launch.

### How to review 
- Check the jsonnet all makes sense
- Check the build script makes sense
- Check the rendered schema matches the prototype / changes in trello.
- Make sure routing and validation work as specified.

Quick-launch: https://rich-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HH&region_code=GB-ENG&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/3184-build-accommodation/data/en/census_household_gb_eng.json

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by donnap573 — APPROVED** [review]

**Comment by MebinAbraham** [comment]
[Introduce HH Schema & Build Accommodation Eng/Wls (M)](https://trello.com/c/5V7lUw1C/3184-introduce-hh-schema-build-accommodation-eng-wls-m)

**Review by iwootten — DISMISSED** [review]
This will need the schemas added to extract/translate schemas. Currently there's no additional strings flagged.

**Comment by bitdivision** [comment]
Very good point @iwootten. I missed that. I've added household_gb_wls to the script now.

**Review by donnap573 — APPROVED** [review]

File: data/en/census_household_gb_eng.json [hunk 1]
```diff
@@ -0,0 +1,544 @@
+{
```
Comment (iwootten) [inline data/en/census_household_gb_eng.json]: I'm guessing this wasn't meant to be committed?

Comment (bitdivision) [reply data/en/census_household_gb_eng.json]: It was, it's used to provide quick-launch. I mentioned it in the PR description but will delete once approved.

Comment (bitdivision) [reply data/en/census_household_gb_eng.json]: Or now, since Donna is happy.

File: scripts/build_schemas.sh [hunk 2]
```diff
@@ -2,37 +2,39 @@
...
     CENSUS_MONTH_YEAR_DATE="2019-10"
 
-    if [[ "$region_code" = "GB-NIR" ]]; then
-        SOURCE_FILE="data-source/jsonnet/northern-ireland/census_individual.jsonnet"
+    CENSUS_TYPES=("individual" "household")
 
-        jsonnet --tla-str region_code="${region_code}" --tla-str census_date="${CENSUS_DATE}" "${SOURCE_FILE}" > "${DESTINATION_FILE}"
-    else
-        SOURCE_FILE="data-source/jsonnet/england-wales/census_individual.jsonnet"
+    for census_type in "${CENSUS_TYPES[@]}"; do
```
Comment (MebinAbraham) [inline scripts/build_schemas.sh]: Can we please make it 
```sh
for census_type in "individual" "household";
```

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**